_projects/ft_workflow_project.md

```diff
@@ -102,26 +102,42 @@ derive a clear model for replication, as there are many ways to implement it,
 even for a single task.
 
 
-## Results for 2016/2017
+## Results
+
```

Comment:
Could you split this to 2016/2017 and 2017/2018? I'm not sure this is needed, just asking for now..

Reply:
Yes I can do that. I will edit the file again.